app/models/dataset.ts

```diff
@@ -77,6 +77,12 @@ export interface Structure {
   schema?: Schema
 }
 
+export interface ColumnProperties {
+  title?: string
+  type?: string
```

Comment:
type can be either a string or array of strings:
```suggestion
  type?: string | string[]
```
We have users that create datasets with multiple column types. Yes, lots of code will break with this assumption, but at least capturing this nuance in the type system is a start